database/tables/issues.pg

```diff
@@ -24,7 +24,7 @@ indexes
     issues_course_id_open_idx: USING btree (course_id, course_caused, open)
     issues_course_instance_id_idx: USING btree (course_instance_id)
     issues_instance_question_id_idx: USING btree (instance_question_id)
-    issues_question_id_open_idx: USING btree (course_id, course_caused, open)
+    issues_question_id_open_idx: USING btree (question_id, open)
```

Comment:
Should we include `course_caused` in this index like we do for `issues_course_id_open_idx`?

Reply:
Yep, the query above selects on `course_caused` so I think we should also index on it.

Reply:
Done in 0a35ca1.